### PR TITLE
Fix form526 form_data

### PR DESCRIPTION
**Pull request by annaswims**
messed up this branch.  fixed in #5651

### CODE REVIEW

File: rakelib/form526.rake [hunk 1]
```diff
@@ -230,4 +230,94 @@ namespace :form526 do
...
+  desc 'Convert SIP data to camel case and fix checkboxes'
+  task convert_sip_data: :environment do |_, _args|
+    def to_olivebranch_case(method, val)
+      OliveBranch::Transformations.transform(
+        val,
+        OliveBranch::Transformations.method(method)
+      )
+    end
+
+    # make a has that has both the original and corrupted versions of the disability name.
```
Comment (edmangimelli) [inline rakelib/form526.rake:243]:
```suggestion
    # make a hash that has both the original and corrupted versions of the disability name.
```

File: rakelib/form526.rake [hunk 2]
```diff
@@ -230,4 +230,94 @@ namespace :form526 do
...
+        end
+        form_data_hash['view:isPow']['powDisabilities'] = new_pow_disability_names
+      end
+      form_data_hash
+    end
+
+    @affected_forms = []
+
+    # forms expire a year after they're last updated by the user so we want to disable updating the updated_at.
+    ActiveRecord::Base.record_timestamps = false
```
Comment (edmangimelli) [inline rakelib/form526.rake:293]: would doing something like
```
...
in_progress_form.save(touch: false) # saves without updating timestamps
```
be a good way to go?
https://bigbinary.com/blog/rails-5-allows-updating-without-updating-timestamps

File: rakelib/form526.rake [hunk 3]
```diff
@@ -230,4 +230,94 @@ namespace :form526 do
...
+    begin
+      # get all of the forms that have not yet been converted.
+      in_progress_forms = InProgressForm.where(form_id: FormProfiles::VA526ez::FORM_ID)
+                                        .where("metadata -> 'return_url' is not null")
```
Comment (edmangimelli) [inline rakelib/form526.rake:297]: dumb question, why would `return_url` ever be `null`?

Comment (annaswims) [reply rakelib/form526.rake:297]: It's really just checking to see whether the snake or camel case key is present.

**Comment by annaswims** [comment]
🤦 git snafu.. I'll have this fixed soon.

**Pull request closed without merging**